### ISSUE TITLE
CROSS JOIN clause is removed on export

### ISSUE THREAD

**Issue by domenk** [phpmyadmin/phpmyadmin#15316]
### Describe the bug

CROSS JOIN clause is removed from the query when trying to export query results to CSV, CSV for MS Excel or OpenDocument Spreadsheet (I haven't tested the other formats). Consequently query fails.

### To Reproduce

1. Create a table:
```sql
CREATE TABLE `crossjoin_test` (
  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
);
```

2. Insert some rows:
```sql
INSERT INTO `crossjoin_test` (`id`) VALUES (NULL), (NULL), (NULL);
```

3. Run query with CROSS JOIN:
```sql
SELECT * FROM `crossjoin_test` AS `t1` CROSS JOIN `crossjoin_test` AS `t2` WHERE `t1`.`id` > 1 AND `t2`.`id` > 1;
```

4. Export query results in CSV, CSV for MS Excel or OpenDocument Spreadsheet format.

5. Exported file (in any of the mentioned formats) contains HTML code for the error. Error mentions that the query is:
```sql
SELECT * FROM `crossjoin_test` WHERE crossjoin_test.`id` > 1 AND `t2`.`id` > 1
```

### Expected behavior

Query should not be rewritten and query results should be normally exported.

### Server configuration

- Operating system: Windows
- Web server: Apache 2.4.39
- Database version: MySQL 8.0.16
- PHP version: 7.3.6
- phpMyAdmin version: 4.9.0.1 (also tested on demo server with master 9102b98)

### Additional context

If CROSS JOIN is replaced with INNER JOIN, export works as expected.

**Comment by williamdes** [comment]
You can also see that the alias is removed, see: #14691

**Comment by williamdes** [comment]
@Tithugues Can you have a look at this one if you want?

**Comment by williamdes** [comment]
Duplicate of #12241

**Comment by williamdes** [comment]
Will be fixed as #12241
